server/views/components/goal-summary-card-v2/template.njk

```diff
@@ -0,0 +1,92 @@
+{% from "govuk/macros/attributes.njk" import govukAttributes %}
+
+  {#- Set classes for this component #}
+  {%- set classNames = "govuk-summary-card goal-summary-card" -%}
+  {%- if params.classes %}
+    {% set classNames = classNames + " " + params.classes %}
+  {% endif %}
+
+  {#- Define component attributes #}
+  {%- set componentAttributes %} class="{{ classNames }}" {{- govukAttributes(params.attributes) -}} {% if params.id %} id="{{ params.id }}"{% endif %}{% endset %}
+
+<div {{ componentAttributes | safe }}>
+  <div class="govuk-summary-card__title-wrapper">
+    <h3 class="govuk-summary-card__title">
+      <img src="/assets/images/icon-goal.svg" role="presentation" alt="" width="35px" height="28px" />
+      Achieve goal by {{ params.goal.targetCompletionDate | formatDate('D MMMM YYYY') }}
+    </h3>
+    {% if params.actions | length %}
+    <ul class="govuk-summary-card__actions goal-summary-card-actions govuk-!-display-none-print">
+      {% for action in params.actions %}
+
+        {% if action['render-if'] == null or action['render-if'] === true %}
+          {#- Set classes for this action #}
+          {%- set classNames = "govuk-link" -%}
+          {%- if action.classes %}
+            {% set classNames = classNames + " " + action.classes %}
+          {% endif %}
+          {%- set actionAttributes %} class="{{ classNames }}" {{- govukAttributes(action.attributes) -}} {% if action.id %} id="{{ action.id }}"{% endif %}{% endset %}
+          <li class="govuk-summary-card__action">
+            <a {{ actionAttributes | safe }} href="{{ action.href }}">{{ action.title | safe }}</a>
+          </li>
+        {% endif %}
+      {% endfor %}
+    </ul>
+    {% endif %}
+  </div>
+  <div class="govuk-summary-card__subtitle-wrapper">
```

Comment:
The reason for a subtitle was that I discovered that a really long goal title sent the archive/update (not currently using v2) and reactivate buttons flying